lib/matplotlib/tests/test_agg.py

```diff
@@ -1,7 +1,6 @@
 import io
 
 import numpy as np
-from numpy.testing import assert_array_almost_equal
```

Comment:
We've been using `from numpy.testing import ...` consistently throughout the library. If you want to change this, it should be changed in all test files.

Moreover, I'm not at fan of a fully qualified `np.testing.assert_array_almost_equal`. That's taking already half of your line if you try to fit on 79 chars. If you don't want bare imports (which I think are ok for the testing functions), you should change to `import numpy.testing as npt` (c.f. the [assert_almost_equal example in the numpy docs]( https://docs.scipy.org/doc/numpy/reference/generated/numpy.testing.assert_almost_equal.html))

Reply:
I agree with @timhoffm here.

Reply:
We're somewhat inconsistent about these, so I don't mind reverting this bit. I do want to get rid of `nptest` used below; that one's very different from everything else.